source/style-guide/writing/write-for-accessibility.txt

```diff
@@ -80,7 +80,7 @@ Write for Ease of Reading
 
 - Break up walls of text to aid in scannability. For example, separate
   paragraphs, create headings, and use lists.
-- Use shorter sentences. Try to use fewer than 26 words per sentence.
+- Use shorter sentences. Try to use fewer than 25 words per sentence.
```

Comment:
**issue:** As originally written, this would be 25 or fewer words. With this change, it would be 24 or fewer words. For clarity, maybe rephrase to "25 or fewer words"?

Reply:
true enough. I'll revert this - though it sounds strange to me.